### PR TITLE
migrate(status-card): css co classes to odf specific

### PR DESCRIPTION
**Pull request by debjyoti-pandit**
image

### CODE REVIEW

File: packages/shared/styles/_common.scss [hunk 1]
```diff
@@ -48,3 +48,12 @@
     margin-bottom: 0 !important;
   }
 }
+
+.odf-dashboard-body {
```
Comment (GowthamShanmugam) [inline packages/shared/styles/_common.scss:52]: instead of common lets move this inside dashboard.scss

Comment (debjyoti-pandit) [reply packages/shared/styles/_common.scss]: Done

File: packages/shared/styles/_dashboard.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.odf-dashboard-body {
```
Comment (bipuladh) [inline packages/shared/styles/_dashboard.scss:1]: Lets create a file named `dashboard.scss` next to `dashboard.tsx` file (same directory) and move this content there. This style might not be useful for every consumer of common.

**Pull request merged**